src/main/java/io/seqera/migtool/MigTool.java

```diff
@@ -321,7 +321,10 @@ else if( locations.startsWith(SOURCE_FILE)) {
         else {
             throw new IllegalArgumentException("Invalid locations prefix: " + locations);
         }
+
         log.debug("Scanned {} migration files, {} amended files and {} fixed files", migrationEntries.size(), amendedEntries.size(), fixedEntries.size());
+        if (amendedEntries.size() != fixedEntries.size())
```

Comment:
This check might not be enough as there could be a file with only a `.amended` version and another file with only a `.fixed` version that would make it pass. I think it might be better to check that each entry has a matching companion (and report which one doesn't have it).